data/live_response.yaml

```diff
@@ -0,0 +1,165 @@
+# Live Response specific artifacts.
+---
+name: LiveResponseApplicationConfigsAndLogs
+doc: Group of configuration files and logs of installed applications.
+sources:
+- type: ARTIFACT_GROUP
+  attributes:
+    names:
+    - MicrosoftIISLogs
+    - MicrosoftSqlServerErrorLogs
+    - RedisConfigFile
+    - TomcatFiles
+    - TomcatPasswordFile
+supported_os: [Windows]
+---
+name: LiveResponseExecution
+doc: Group of process/command execution related artifacts.
```

Comment:
@binglot let's chate more about the naming. Would it be clearer to name them according to their purpose e.g. TriageWindowsProcessExecution ?

Reply:
Also these groups do not necessary have to be used for live response (or triage for that matter).

Reply:
As discussed offline, I've changed it from 'Live Response' to 'Triage'.